keyboards/splitkb/aurora/helix/rev1/rev1.c

```diff
@@ -0,0 +1,56 @@
+/* Copyright 2023 splitkb.com <support@splitkb.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+//  Overriding is_keyboard_left() in qmk_firmware/quantum/split_common/split_util.c to limit the handedness check only once.
+//  reason: SPLIT_HAND_PIN is shared with the RGB pin so we don't want to keep reusing the pin for handedness checks.
+bool is_keyboard_left(void) {
```

Comment:
Since I suspect it's too late here ... but it looks like there are a number of unpopulated matrix positions on both sides... the split matrix handedness could have been used here instead of the split pin.
https://docs.qmk.fm/#/feature_split_keyboard?id=handedness-by-matrix-pin

Reply:
I’m aware of the feature, but using matrix pins for side detection seems to keep ZMK awake, draining the battery in the process.

Reply:
Most likely, that's because it's using interrupts, and doesn't sleep until all keys are released.  But using polling instead of interrupts is probably less power efficient, as well.  I'm surprised that zmk doesn't have a way to ignore it in certain contexts.

Reply:
Just as a heads up, this tweak will likely not be needed in the future, re: #21875.

Reply:
Good to know, but it will be needed for now then.